torchvision/transforms/functional.py

```diff
@@ -124,7 +124,7 @@ def _is_numpy_image(img: Any) -> bool:
     return img.ndim in {2, 3}
 
 
-def to_tensor(pic: Union[PILImage, np.ndarray]) -> Tensor:
+def to_tensor(pic: Union[PILImage, np.ndarray[int, int]) -> Tensor:
```

Comment:
@adamtheturtle thanks for the PR - just to make sure I understand correctly, does `np.ndarray[int, int]` mean the ndarry is expected to only have 2 dimensions? If that's the case then that is not correct as we allow both 2D and 3D arrays.